HQSmokeTests/testPages/users/mobile_workers_page.py

```diff
@@ -171,6 +190,29 @@ def click_create(self):
         assert self.username == new_user_created, "Could find the new mobile worker created"
         print("Mobile Worker Created")
 
+    def check_for_group_in_downloaded_file(self):
+        self.wait_to_click(self.click_group_menu)
+        self.wait_to_clear_and_send_keys(self.group_name, self.created_group)
+        self.wait_to_click(self.add_group_button)
+        self.send_keys(self.users_drop_down, "username_123")
+        self.wait_to_click(self.select_user)
+        self.wait_to_click(self.update_button)
+        group_id_value = self.driver.current_url.split("/")[8]
+        print(group_id_value)
+        time.sleep(2)
+        self.download_mobile_worker()
+        newest_file = latest_download_file()
+        path = os.path.join(UserData.DOWNLOAD_PATH, newest_file)
+        data = pd.read_excel(path, sheet_name='groups')
+        df = pd.DataFrame(data, columns=['id'])
+        assert group_id_value in df['id'].values, "Group is not present"
```

Comment:
After getting the group id, only this bit is relevant and this function could be called after 
https://github.com/dimagi/dimagi-qa/blob/e360c8546c5457d1e657c1e3c8ae5c006894eebe/HQSmokeTests/testCases/test_02_users.py#L56

Reply:
https://github.com/dimagi/dimagi-qa/pull/115/commits/4e0b0011db3f36e27ef401516153ace30a1351aa